request.ts

```diff
@@ -111,21 +109,25 @@ export class PinejsClientRequest extends PinejsClientCore<
 						},
 					);
 				})
-				.catch(BluebirdLRU.NoSuchKeyError, () => {
-					return requestAsync(params).then(
-						({ statusCode, body, headers }): CachedResponse => {
-							if (200 <= statusCode && statusCode < 300) {
-								return {
-									etag: headers.etag as string,
-									body,
-								};
-							}
-							throw new StatusError(body, statusCode);
-						},
-					);
+				.catch((err) => {
+					if (err instanceof getBluebirdLRU().NoSuchKeyError) {
+						return requestAsync(params).then(
+							({ statusCode, body, headers }): CachedResponse => {
+								if (200 <= statusCode && statusCode < 300) {
+									return {
+										etag: headers.etag as string,
+										body,
+									};
+								}
+								throw new StatusError(body, statusCode);
+							},
+						);
+					}
+					throw err;
 				})
 				.then((cached) => {
 					this.cache!.set(params.url, cached);
+					const { cloneDeep } = require('lodash') as typeof import('lodash');
```

Comment:
Maybe you could also just require `lodash/cloneDeep`, but I guess that the lodash require in here will already be near 0 cost, since it would have been loaded elsewhere.

Reply:
Yeah, the expectation is that this module would only be used in nodejs and so requiring lodash/cloneDeep would actually be worse (especially if anywhere else has already imported 'lodash' directly)